sycl/include/sycl/ext/intel/experimental/esimd/detail/simd_view_impl.hpp

```diff
@@ -463,7 +434,7 @@ class simd_view_impl {
   /// \return replicated simd instance.
   template <int Rep, int VS, int W, int HS>
   get_simd_t<element_type, Rep * W> replicate(uint16_t OffsetX) {
```

Comment:
@kbobrovs, should simd_view::replicate methods naming be consistent with simd::replicate?
Also, it looks like simd::replicate method called on line 413 does not exist.

Reply:
yes, they should, thanks for catching. could you please fix that too?

Reply:
As part of this PR? Or in a separate PR?

Reply:
either way is OK for me